packages/core/nuxt-theme-module/theme/components/CartSidebar.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <div id="cart">
     <SfSidebar
-      :visible="isCartSidebarOpen"
+      :visible="true"
```

Comment:
Wouldn't it be cleaner to just have `visible` by default?

Reply:
@AdamPawlinski good idea to have all modals/sidebars visible by default ;)